src/components/Chat/Chat.stories.tsx

```diff
@@ -142,55 +144,54 @@ const BuildMessages: (props: { children?: ReactNode }) => JSX.Element = ({
       },
     ]);
   }, [addMessage, setQuickReplies]);
-
-  return children as JSX.Element;
 };
 
+const Wrapper = ({ children }: { children: ReactNode }) => {
+  useMessages()
+  return children as JSX.Element
+}
+
+const ModalContainer = styled.div`
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  background-color: rgba(12, 12, 12, 0.65);
+  transform: translate(-50%, -50%);
+  width: 800px;
+  height: 600px;
+  padding: 1rem;
+  max-height: 100vh;
+  max-width: 100vw;
+  border: 1px solid black;
+`
+
+const FullscreenContainer = styled.div`
+  height: 100vh;
```

Comment:
Scrollbar displayed ... Adding an normalize.css in storybook/config.js may solve problem but i don't know if it is a good idea?